src/sections/sticky.html

```diff
@@ -0,0 +1,3 @@
+<div class="flex justify-center mt-20">
+  <img src="../src/images/sticky-shape.svg" alt="" />
```

Comment:
Try to fill the alt attribute with the image description.
example:
`<img src="../src/images/sticky-shape.svg" alt="Yellow shape" />`